sceptre/plan/actions.py

```diff
@@ -1152,3 +1152,10 @@ def dump_config(self, config_reader: ConfigReader):
         """
         stack_path = normalise_path(self.stack.name + ".yaml")
         return config_reader.read(stack_path)
+
+    @add_stack_hooks
+    def dump_template(self):
+        """
+        Returns the Template for the Stack
+        """
+        return self.stack.template.body
```

Comment:
This is identical to `generate()`.

Reply:
Yes, it needs to be identical to `generate()` since the latter is deprecated and would be removed. Also, the naming standard requires that CLI commands have a method named after the CLI command (at least I hope we preserve and continue that naming standard!).

Reply:
If we agree to alias `generate` and `dump template` of course then perhaps we only need one method here.

Reply:
Yeah, I would say that you should make `generate` here call this method and return its result. That should be a pretty harmless way to start aliasing.

Reply:
ok done

Reply:
If you're deprecating `generate`, you definitely should do that correctly, both in docs and using the `deprecated` decorator.

Reply:
ok done